src/ECER.Engines.Transformation/InviteLinks/IInviteLinkTransformationEngine.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ECER.Managers.Admin.Contract.InviteLinks;
+
+namespace ECER.Engines.Transformation.InviteLinks;
+
+/// <summary>
+/// transforms references links
+/// </summary>
+public interface IInviteLinkTransformationEngine
+{
+  Task<ManageInviteLinkCommandResponse> Transform(ManageInviteLinkCommand command);
```

Comment:
the engine here deserves its own contract - the naming here smells. A better way would be:

Task<TransformResponse> Transform(TransformRequest request)

abstract record TransformRequest
record EncryptInviteLinkRequest : TransformRequest
record EncryptedIinkResponse: TransformResponse